jOOQ-examples/jOOQ-spring-boot-example/src/main/java/org/jooq/example/spring/repository/BookRepository.java

```diff
@@ -0,0 +1,22 @@
+package org.jooq.example.spring.repository;
+
+import org.jooq.Configuration;
+import org.jooq.DSLContext;
+import org.jooq.example.db.h2.tables.daos.BookDao;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class BookRepository extends BookDao {
+
+  private final DSLContext dslContext;
+
+  public BookRepository(Configuration configuration, DSLContext dslContext) {
```

Comment:
This generator must be required.

or Genreated method in dao will throw ```org.jooq.exception.DetachedException``` like below

<img width="1009" alt="image" src="https://github.com/jOOQ/jOOQ/assets/50672087/91262d27-63f7-4a6e-ba23-82e8aa0bc9b2">